CHANGELOG.md

```diff
@@ -0,0 +1,63 @@
+# Change log
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](http://keepachangelog.com) and this project adheres to [Semantic Versioning](http://semver.org).
+
+## 0.3.6 - 2017-03-04
+### Added
+- Add CHANGELOG.md.
```

Comment:
Sure. I will update all these change today. Had some personal stuff for the past two day.